compiler/src/typed/typed_well_formedness.re

```diff
@@ -276,6 +277,53 @@ module WellFormednessArg: TypedtreeIter.IteratorArgument = {
             Grain_parsing.Location.prerr_warning(exp_loc, warning);
           };
         }
+      // Check: Warn if using Pervasives print on WasmXX types
+      | TExpApp(
+          {
+            exp_desc:
+              TExpIdent(
+                Path.PExternal(Path.PIdent({name: "Pervasives"}), "print"),
```

Comment:
Why didn't you just do "print" | "toString"

Reply:
Because the error is different. and I wouldn't have had a reference to what the given function was to know which error to give.